regulations/static/regulations/js/source/helpers.js

```diff
@@ -38,6 +38,17 @@ if (!Array.prototype.indexOf) {
   };
 }
 
+$(window).load(function () {
+
+  // notice and comment homepage expandable section
+  $('.nc-homepage .expandable .button').click(function (){
+    $(this).next().fadeToggle();
```

Comment:
I have the same concern here as in #352. This binds the logic very closely to the markup, which means that at some point we'll modify the markup, things will break, and we won't know why.